libs/elements/layout/images/src/lib/components/elewa-image-container/elewa-image-container.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { ImageConfig } from '../../../../../../../models/schema/ui/images/image-config.interface';
+
+@Component({
+  selector: 'elewa-website-elewa-image-container',
+  templateUrl: './elewa-image-container.component.html',
+  styleUrls: ['./elewa-image-container.component.scss'],
+})
+export class ElewaImageContainerComponent {
+  pill: ImageConfig = {
+    title: 'Pill',
+    imageSrc: 'https://media.gettyimages.com/id/1346973410/photo/young-woman-at-a-library.jpg?s=612x612&w=0&k=20&c=PW6KYwtS3oUx1mPcbfyqDcZotnyjFO-cwfbOsyG2ERk=',
+    visualisation: 0,
+  };
+  stacked: ImageConfig = {
+    title: 'Stacked',
+    imageSrc: 'https://media.istockphoto.com/id/1178524829/photo/young-woman-making-choice-donut-or-apple.jpg?s=612x612&w=0&k=20&c=WeZ6-DQ8npDAhDNDZkcE48gtFFbqfTIIvpAfSfsh0Uo=',
+    visualisation: 1,
```

Comment:
Dummy data should be stored in the data module under sections but that was not required for this issue since it's a reusable component. But if needed always save under a data module